dataline-workers/src/main/java/io/dataline/workers/singer/SingerSyncWorker.java

```diff
@@ -22,9 +22,8 @@
  * SOFTWARE.
  */
 
-package io.dataline.workers;
+package io.dataline.workers.singer;
```

Comment:
Renamed it to SingerSyncWorker because it can only work with singer messages

Reply:
true given the changes you made to the constructor, but why are you changing the implementation for that to be the case? What in this class makes it important to pass in a SingerTap and a SingerTarget as opposed to just a Tap and Target?

Reply:
discussed offline: the DefaultSyncWorker is Singer protocol specific but not specific to a singer tap and target. This will get renamed to something like SingerProtocolSyncWorker and will take in SingerProtocolTap and SingerProtocolTarget.